__docs__/wonder-blocks-dropdown/combobox.stories.tsx

```diff
@@ -192,31 +195,49 @@ export const ControlledCombobox: Story = {
         const [opened, setOpened] = React.useState(args.opened);
         const [value, setValue] = React.useState(args.value);
 
+        React.useEffect(() => {
+            setOpened(args.opened);
+        }, [args.opened]);
+
```

Comment:
**note:** refactored this story to open the controlled programmatically via the SB `play` feature. There was an odd issue where the combobox was opened on load, then to focus would move to this combobox instead of keeping the scroll at the top of the page. You can see the repro here: https://khan.github.io/wonder-blocks/?path=/docs/packages-dropdown-combobox--docs

Now that I switched to play, Chromatic still snapshots the story when it is opened, but we don't see it opened in the Docs page (which I think it makes sense).